examples/data_sharing.tex

```diff
@@ -0,0 +1,210 @@
+\documentclass[a4paper]{article}
+\usepackage{data}
+
+%%%%% Now we input our variable terms as glossary entries %%%%%
+
+\newglossaryentry{type}{
+	name={Company},
+	first={the ``Company''},
+	text={Company},
+	description={is a commercial undertaking}
+}
+
+\newglossaryentry{undertaking}{
+	name={King of the Eagles},
+	first={King of the Eagles, hereafter `\glsentrytext{type}'},
+	text={the \glsentrytext{type}},
+	description={are Big birds. Live on a hill}
+}
+
+\newglossaryentry{processor}{
+	name={Mirkwood Elves},
+	first={Mirkwood Elves, hereafter `the Processor'},
+	text={Data Processor},
+	description={are the Elves of the Noldor Line}
+}
+
+\newcommand{\services}{Orc-slaying}
+
+\newcommand{\companyAddress}{123 Eirie Road}
+
+
+\newcommand{\processorAddress}{Third Tree, Mirkwood}
+
+\newcommand{\processorEmail}{elves@place.mw}
+
+\title{Data Sharing Agreement between \Glsentrytext{undertaking} \& \Glsentrytext{processor}}
+
+\begin{document}
+
+\begin{titlepage}
```

Comment:
i was going to say wrap the titlepage into a `\maketile` command in .sty but i noticed there isn't one. @bond7o6 do we want to leave this as is (and clean up in #30) or make a .sty here?

Reply:
I'd say using `\maketitle` here would be easier for later, but it isn't to big an issue to fix later. So if everything else gets addressed except this, I'd say just merge and I'll clean it up in the class one.